### ISSUE TITLE
PCB export issue

### ISSUE THREAD

**Issue by msasdk** [AislerHQ/PushForKiCad#22]
Hello!
I have created this PCB in KiCad 7.0:
[problematic.zip](https://github.com/AislerHQ/PushForKiCad/files/10929157/problematic.zip)
![image](https://user-images.githubusercontent.com/13538204/223958515-38d5ee4e-e54f-45f3-bd9b-582fa4f790ce.png)
![image](https://user-images.githubusercontent.com/13538204/223958582-465fc1d4-c3f2-466b-b9c7-cbd17a9b1587.png)

When exported as gerber, things looks as they are intended:
![image](https://user-images.githubusercontent.com/13538204/223960039-5478b20a-eb16-4409-90d3-49abf4d225cc.png)
However, when exported using the Aisler plugin, or if the kicad_pcb fileis imported directly on Aisler.net, I get following result.

 ![image](https://user-images.githubusercontent.com/13538204/223958670-250dcb08-69f2-4228-9712-a0e85293e90b.png)

**Comment by PatrickFranken** [comment]
Hi! I guess this issue is more KiCad related. If I open the generated gerber files using the Ucamco reference viewer I get a list of issues. These issues also cause our CAM to misbehave.
![Screenshot from 2023-03-14 15-23-42](https://user-images.githubusercontent.com/8692465/225031926-821417ef-8434-4bf5-86ce-6fe9a5652451.png)

**Comment by msasdk** [comment]
But as stated in the issue description, this does not occur if I generate the gerber with KiCad without the plugin.

**Comment by msasdk** [comment]
These gerbers are generated directly from KiCad
[grb.zip](https://github.com/AislerHQ/PushForKiCad/files/10969840/grb.zip)

**Comment by PatrickFranken** [comment]
You are right on this! However I have no clue what causes KiCad to plot these corrupted Gerbers as the plugin actually uses the same interface as the UI. I'll check with the KiCad developers!

**Comment by Golany2806** [comment]
Seems there is a problem using push to aisler using kicad...it connecting to microsoft edge without loading the files...
